src/main/java/com/asem/regestration/model/dao/CourseDAO.java

```diff
@@ -0,0 +1,15 @@
+package com.asem.regestration.model.dao;
+
+import com.asem.regestration.model.entity.Courses;
+import com.asem.regestration.model.entity.Student;
+
+import java.sql.SQLException;
+
+public interface CourseDAO {
+
+    Courses findById(int id) throws SQLException;
+    void deleteCourse(int id) throws SQLException;
+    void creat(Courses courses) throws SQLException;
```

Comment:
type creat -> create

Reply:
Its better to be consistent.
deleteCourse
createCourse
updateCourse

or

delete
update
create